base/markdown/render/plain.jl

```diff
@@ -37,6 +37,46 @@ function plain(io::IO, list::List)
     end
 end
 
+function plain(io::IO, md::Table)
+    col_widths = reduce(max, map(cell -> map(ansi_length, cell), md.rows))
+
+    for (n, row) in enumerate(md.rows)
+        for (i, h) in enumerate(row)
+            # TODO use not terminal version of print_align
+            if i != 1
```

Comment:
It's quite common in Julia to write things like this as `i == 1 || print(io, " | ")`.